chapters/methodology.tex

```diff
@@ -14,3 +14,25 @@ \subsection{State Machine}
 \subsection{User Interface}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
+\subsection{Model Training}
+
+As stated in the background section, soHappy uses a deep learning model to 
+provide information about a person smiling by utilizing TensorFlow.
+
+However, for TFLite to detect smiling, the model must be trained first. This

```

Comment:
Maybe "a model" instead of "the model", because a model is the _result_ of training - you don't start off with an empty model and train that.